src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -0,0 +1,349 @@
+    })
+  }
+
+  handleAccordionItemKeyPress = (event, index) => {
+    if (event.key === 'Enter' || event.which == 13 || event.keyCode == 13) {
+      this.toggleAccordionItem(index)
+    }
+  }
+
+  render() {
+    const {
+      LinkComponent,
+      hideMobileCta,
+      hideDesktopCta,
+      logoHref,
+    } = this.props
+
+    const renderCtaButton = (showWhenScrolled) => (
+      <a
+        className={
+          'cta-button ' + (showWhenScrolled ? 'show-when-scrolled' : '')
+        }
+        onClick={this.props.trackCtaClick}
+        href={LINKS.TERM.href}
+      >
+        <Button.Small.BlackOutline>Check my price</Button.Small.BlackOutline>
+      </a>
+    )
+
+    const { showMobileMenu, activeAccordionItem } = this.state
+    console.log(this.state)
+    const renderTextLink = (link) => (
+      <div key={link.id} className={styles.textLink}>
+        <NavLink
+          href={link.href}
+          LinkComponent={link.href !== '/login/' ? LinkComponent : null}
+        >
+          {link.title}
+        </NavLink>
+      </div>
+    )
+
+    // These are bespoke icons but design may replace w/FA at a later date
+    const renderSearchIcon = (link) => {
+      return (
+        <NavLink
+          className={styles.searchIcon}
+          key={link.id}
+          href={link.href}
+          LinkComponent={link.href !== '/login/' ? LinkComponent : null}
+        >
+          <SearchIcon />
+        </NavLink>
+      )
+    }
+
+    const renderAccountIcon = (link) => {
+      return (
+        <NavLink
+          className={styles.accountIcon}
+          key={link.id}
+          href={link.href}
+          LinkComponent={link.href !== '/login/' ? LinkComponent : null}
+        >
+          <AccountIcon />
+        </NavLink>
+      )
+    }
+
+    const Hamburger = () => (
+      <div className={styles.hamburger}>
+        <TransformingBurgerButton
+          showMobileMenu={showMobileMenu}
+          clickHandler={this.toggleHamburger}
+          keyPressHandler={this.handleHamburgerKeyPress}
+        />
+      </div>
+    )
+
+    return (
+      <div className={styles.blockNavbar}>
+        <div className={styles.navbar}>
+          <Layout.ScrollDetector>
+            <Hamburger />
+            <div
+              className={
+                showMobileMenu
+                  ? [styles.phoneAndTablet, styles.visible].join(' ')
+                  : styles.phoneAndTablet
+              }
+            >
+              <div
+                className={
+                  showMobileMenu ? styles.mobileMenu : styles.hideMobileMenu
+                }
+              >
+                <NavLink
+                  href={logoHref}
+                  LinkComponent={LinkComponent}
+                  className={styles.phoneLogo}
+                >
+                  {LogoWhite({ className: styles.logo })}
+                </NavLink>
+                <Hamburger />
+                <div className={styles.accordion}>
```

Comment:
Can we split out into a separate component, this component is getting big

Reply:
yes should be able to

Reply:
marked in TODOs for fastfollow